File: lua/test/exists_test.lua
-- BluefinTecsMerchantPortal SDK exists test

local sdk = require("bluefin-tecs-merchant-portal_sdk")

describe("BluefinTecsMerchantPortalSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// BluefinTecsMerchantPortal SDK exists test

require_once __DIR__ . '/../bluefintecsmerchantportal_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = BluefinTecsMerchantPortalSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
